protos/sources/vega/governance.proto

```diff
@@ -475,3 +485,69 @@ message Vote {
   // The weight of the vote compared to the total amount of equity-like share on the market.
   string total_equity_like_share_weight = 7;
 }
+
+message CancelTransfer {
+  // Configuration for a governance transfer cancellation.
```

Comment:
Is this for a governance proposal to cancel a transfer?

Reply:
this is for cancelling an enacted governance transfer.

Reply:
Alright so - it's cancelling an enacted transfer that was initiated through governance? A "governance transfer" sounds to me like transferring governance, rather than transferring assets

Reply:
it's governance-initiated transfer